tests/app/public_contracts/schemas/GET_notifications_return.json

```diff
@@ -14,7 +14,14 @@
     },
     "links": {
       "type": "object",
-      "additionalProperties": false
```

Comment:
I will update to include 'prev' based on your comment below. I suppose in future if we add more we can update accordingly...